massa-execution-worker/src/context.rs

```diff
@@ -411,7 +411,14 @@ impl ExecutionContext {
             data.push(1u8);
         }
         // hash the seed to get a unique address
-        let address = Address::SC(SCAddress(massa_hash::Hash::compute_from(&data)));
+        let address = Address::SC(
+            SCAddress::from_bytes_without_version(
+                0,
+                massa_hash::Hash::compute_from(&data).to_bytes(),
+            )
+            .expect("Could not create the sc address"),
+        );
+        //let address = Address::SC(SCAddress::from_bytes_without_version(1, &data)));
```

Comment:
Dead code?

```suggestion
```

Reply:
This will get resolved once we have a "switch" from one version to another. It's mainly to show what is the expected input of the SCAddressV0 and SCAddressV1 constructor.